packages/feeds-client/src/event-handlers/feed/handle-follow-updated.test.ts

```diff
@@ -0,0 +1,174 @@
+import { describe, it, expect, beforeEach } from 'vitest';
+import { Feed } from '../../Feed';
+import { FeedsClient } from '../../FeedsClient';
+import { handleFollowUpdated } from './handle-follow-updated';
+import {
+  generateFollowResponse,
+  generateFeedResponse,
+  getHumanId,
+  generateOwnUser,
+} from '../../test-utils/response-generators';
+import { FollowResponse } from '../../gen/models';
+import { shouldUpdateState } from '../../state-updates/state-update-queue';
+
+describe(handleFollowUpdated.name, () => {
```

Comment:
Here's what I had in mind. We could even create a [test fixture](https://vitest.dev/guide/test-context.html#extend-test-context) that'd make the setup much easier and highly reusable.